test/gtest/common/test.h

```diff
@@ -48,7 +48,7 @@ class test_base {
 
     void get_config(const std::string& name, std::string& value,
                             size_t max);
-    virtual void set_config(const std::string& config_str);
+    virtual void set_config(const std::string& config_str = "");
```

Comment:
Had to add it, otherwise it won't compile. `UCS_TEST_F` macro may expand to a call to `set_config()` (without args).

Reply:
An alternative would be to add an overload `set_config() {}`, it might be cleaner but less efficient.